app/Libraries/Search/BeatmapsetSearchParams.php

```diff
@@ -13,6 +13,27 @@ class BeatmapsetSearchParams extends SearchParams
     const PLAYED_STATES = ['played', 'unplayed'];
     const STATUSES_NO_CACHE = ['favourites', 'mine'];
 
+    /** @var array|null */
+    public $ar;
+
+    /** @var artist|null */
```

Comment:
`array`? `string`? But it's assigned the value from an `array_merge`, so...
Same with `$creator`